build-logic/src/main/kotlin/terasology-metrics.gradle.kts

```diff
@@ -93,9 +93,6 @@ configure<PmdExtension> {
 }
 
 configure<SpotBugsExtension> {
-    // The version of the spotbugs tool https://github.com/spotbugs/spotbugs
-    // not necessarily the same as the version of spotbugs-gradle-plugin
-    toolVersion.set("4.7.0")
```

Comment:
Why do remove this instead of updating it?

Reply:
setting it gave an unexpected error, and if one NEEDS to anyway update it with a different gradle version, better remove it. if somebody feels to set the version again for a yet unknown reason, it is not too much work to add a line again.

Reply:
test with
```
gradle clean spotbugsMain
....

❯ ls -l engine/build/reports/spotbugs/main.xml 
-rw-r--r-- 1  4585709 21. Aug 08:28 engine/build/reports/spotbugs/main.xml

```

Reply:
ah okay, wasn't aware it was overriding the default version.
generally I'd rather know which version we're using, but considering our track record of updating dependencies, I guess it's fine to just go with whatever default the latest consumed gradle version implies :+1: